CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
 
+## [1.1.5] - 2023-03-29
+### Changed
+
+* Optimized thread locks and expiring cache for the Enhanced Monitoring Plugin. ([PR #356](https://github.com/awslabs/aws-mysql-jdbc/pull/356)) ([PR #373](https://github.com/awslabs/aws-mysql-jdbc/pull/373))
+* Added checking log level to avoid unnecessary resource allocation. ([PR #367](https://github.com/awslabs/aws-mysql-jdbc/pull/367))
+* Updated README.md with the following
+  * Improvements to the `clusterInstanceHostPattern` Failover Parameter example. ([Link](https://github.com/awslabs/aws-mysql-jdbc#failover-parameters))
+  * Latest Maven Central URLs.([Link #1](https://github.com/awslabs/aws-mysql-jdbc#direct-download-of-the-jar-file), [Link #2](https://github.com/awslabs/aws-mysql-jdbc#as-a-maven-dependency), [Link #3](https://github.com/awslabs/aws-mysql-jdbc#as-a-gradle-dependency)) 
```

Comment:
Would it make sense to change this to the following to avoid mystery Links with #'s:

 * Latest Maven Central URLs for [direct download of the jar file](https://github.com/awslabs/aws-mysql-jdbc#direct-download-of-the-jar-file), [as a maven dependency](https://github.com/awslabs/aws-mysql-jdbc#as-a-maven-dependency), and [as a gradle dependency](https://github.com/awslabs/aws-mysql-jdbc#as-a-gradle-dependency)

Reply:
done